src/main/java/spireMapOverhaul/zones/Junkyard/cards/Bargain.java

```diff
@@ -0,0 +1,106 @@
+package spireMapOverhaul.zones.Junkyard.cards;
+
+import com.badlogic.gdx.Gdx;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.CardStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractSMOCard;
+import spireMapOverhaul.zones.Junkyard.actions.BuyCardAction;
+import spireMapOverhaul.zones.Junkyard.monsters.Peddler;
+
+import java.util.ArrayList;
+
+public class Bargain extends AbstractSMOCard {
```

Comment:
```suggestion
@NoPools
public class Bargain extends AbstractSMOCard {
```
This probably doesn't have to be in the compendium